docs/source/en/model_doc/auto.mdx

```diff
@@ -286,6 +286,14 @@ The following auto classes are available for the following audio tasks.
 
 [[autodoc]] TFAutoModelForSpeechSeq2Seq
 
+### AutoModelForSpeechToSpeech
+
+[[autodoc]] AutoModelForSpeechToSpeech
+
+### AutoModelForTextToSpeech
+
+[[autodoc]] AutoModelForTextToSpeech
```

Comment:
Those auto classes can be added in a separate PR, once we have several models with the same API. Let's focus the PR on the addition of SpeechT5